flows/zeropoint.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Plotting utilities.
```

Comment:
Please give a better docstring!

Reply:
added

Reply:
[https://github.com/rhandberg/flows/pull/10/commits/041e327b79fba812876736771836fa1380d32e43](With this commit)